docs/capi-vsphere.md

```diff
@@ -0,0 +1,178 @@
+# Cluster API - VMware
+
+This example demonstrates how k0smotron can be used with CAPV (Cluster API Provider vSphere).
+
+## Preparations
+
+Before starting this example, ensure that you have met the [general prerequisites](capi-examples.md#prerequisites).
+
+### Prepare the development/deployment server for ClusterAPI
```

Comment:
@nekwar What is the development/deployment server for CAPI?
The docs do look different to the once we have for e.g. CAPO or CAPA. Can you have a look into them and try to stay in the same schema?

Reply:
CAPO and CAPA docs does not say anything about this "development/deployment" server from where `clusterctl` and `kubectl` commands should be executed. IMO, CAPO and CAPA docs should have this info, rather than removing it from CAPV docs

But I agree that the naming ("development/deployment") is a bit confusing, I will change it to "Environment configuration for clusterctl"

Additionally, CAPV docs could not look similar to CAPO and CAPA docs, because there are a lot of differences between platforms.